app/assets/stylesheets/people.scss

```diff
@@ -45,3 +45,11 @@
     .btn-danger { margin-top: 9px; }
   }
 }
+#sharing_message {
```

Comment:
- Avoid using id selectors
- Selector `sharing_message` should be written in lowercase with hyphens